tools/publish-aos.js

```diff
@@ -1,54 +1,137 @@
 const Arweave = require('arweave');
 const path = require('path');
 const fs = require('fs');
+const { TurboFactory } = require('@ardrive/turbo-sdk');
+const { createData, ArweaveSigner } = require('@dha-team/arbundles');
 
 const srcCodeTxIdPlaceholder = '__INSERT_SOURCE_CODE_ID__';
 
 const arweave = Arweave.init({
   host: 'arweave.net',
   port: 443,
   protocol: 'https',
+  logging: false,
```

Comment:
does this do anything?

Reply:
I think so, but it doesnt do it properly anyways - removed